contents/docs/self-host/postgres-vs-clickhouse.md

```diff
@@ -4,27 +4,27 @@ sidebar: Docs
 showTitle: true
 ---
 
-## ClickHouse
+Long-term we think PostHog will have to run on [Clickhouse](https://clickhouse.tech) instead of Postgres as the main database to handle larger volumes of events and more robust analytics capabilities.
 
-> We **strongly recommend ClickHouse** as the most futureproof deployment option. In future, we may remove support for Postgres
+> We **strongly recommend ClickHouse** as the most futureproof deployment option. In future, we may remove support for Postgres.
+
+## I'm new to PostHog, what do I need to know?
+
+All the [deployment instructions](/docs/self-host/deploy/overview) have already been upgraded, so just head over there to get started.
 
-[ClickHouse](https://clickhouse.tech) is an open-source, Apache 2.0 licensed OLAP database. It is very fast at doing analytical queries on large volumes of data. It also stores this data much more efficiently than Postgres does, and we've seen a 70% reduction in the amount of disk space needed to store the same data.
 
-## Postgres
+## I have an existing PostHog (Postgres) instance, what does this mean for me?
 
-A standard instance of PostHog using PostgreSQL is pretty efficient for storing large volumes of data. We've built Posthog so that ingesting a high volume of events should not be a problem even with lower tier Postgres instances.
+We are currently supporting both Postgres and ClickHouse backed instances, however some of the new features are only available on ClickHouse.
 
-However, once you have more data, you might start having issues while running queries. The issue is that Postgres isn't very good at doing analytical queries on larger datasets. This is where ClickHouse comes in.
+We're working on creating an easy migration process, but it's not ready yet. If you're ok with starting from scratch or are experiencing scaling problems we encourage you to move over now. See the [deployment instructions](/docs/self-host/deploy/overview) for spinning up a new instance backed by ClickHouse.
```

Comment:
How can a user of PostHog + Postgres follow along with this and get updates? Do we have a public issue?

Reply:
I propose we just link to https://hub.qovery.com/guides/engineering/how-to-migrate-posthog-self-hosted-to-posthog-cloud/ It talks about moving from self hosted -> cloud ; but that should work for any migration. Note that obviously it doesn't migrate dashboards etc + there's some differences in how auto capture is stored which I'm not sure if we want to handle.

Reply:
```suggestion
We're [working on creating an easy migration process](https://github.com/PostHog/posthog.com/issues/1892), but it's not ready yet. If you're ok with starting from scratch or are experiencing scaling problems we encourage you to move over now. See the [deployment instructions](/docs/self-host/deploy/overview) for spinning up a new instance backed by ClickHouse.
```